app/src/main/java/org/mozilla/focus/widget/ResizableKeyboardViewDelegate.java

```diff
@@ -147,4 +150,13 @@ private int calculateDifferenceBetweenHeightAndUsableArea() {
 
         return delegateView.getResources().getDisplayMetrics().heightPixels - rect.bottom;
     }
+
+    private void updateDynamicViewsVisibility(int visibility) {
+        if (!arrayOfViewsToHide.isEmpty()) {
+            for (Integer viewId : arrayOfViewsToHide) {
+                View viewToHide = delegateView.findViewById(viewId);
+                viewToHide.setVisibility(visibility);
```

Comment:
Should we check if this is nullable before calling this?

Kotlin has changed the way I think about views. :)

Reply:
Yep, we should. Thanks!